tests/framework/CodeInterfaceTests/MOOSEparser/parser_moose/1/out~MOOSE_parser.csv

```diff
@@ -0,0 +1,4 @@
+time,average_fuel_temperature,average_graphite_temperature,delta_temp,total_heat_source
+0,0,0,0,0
+0.001,264.95985083426,202.02485982188,62.934991012381,5540.0527538609
+0.002,265.15753682374,202.17015011614,62.987386707599,5545.5872720965
```

Comment:
I do not recall. Is this file generated by the Code Interface or by MOOSE itself? (In the first case, it should be removed)